### DISCUSSION TITLE
How to setup email DNS for third parties

### DISCUSSION THREAD

**Discussion by Ennyolar96** [vercel/community#4912] [Help]
### Summary

I added vercel DNS to a third parties, so I need to set up the DNS for the email as well so I can have the email of my domain name

### Steps to Reproduce

I need help on that

**Comment by swarnava** [comment]
Hi,

If you use Vercel Nameserver and host domain via Manage dashboard, you should be able to visit Domain configuration via Vercel Dashboard: https://vercel.com/docs/projects/domains/working-with-domains#using-email-with-domains
